src/io/stdin.rs

```diff
@@ -46,6 +47,11 @@ pub fn stdin() -> Stdin {
 #[derive(Debug)]
 pub struct Stdin(Mutex<State>);
 
+#[derive(Debug)]
+pub struct StdinLock<'a>(std::io::StdinLock<'a>);
+
+unsafe impl Send for StdinLock<'_> {}
```

Comment:
@stjepang I changed only StdinLock.
How about such an implementation ❓

Reply:
Looks good! The only thing missing here is a doc comment now I believe!